components/centraldashboard/public/components/main-page.pug

```diff
@@ -1,44 +1,40 @@
+                    paper-item.menu-item [[item.text]]
         aside.flex
         footer.footer
             section.privacy Privacy
-            section.build build version 
-                span(title="Build: v[[buildVersion]] | Dashboard: v[[dashVersion]]") v[[buildVersion]]
+            section.build build version
+                span(title="Build: v[[buildVersion]] | Dashboard: v[[dashVersion]]")  v[[buildVersion]]
     app-header-layout(fullbleed)
-        app-header(slot='header', fixed)
+        app-header(slot='header')
             app-toolbar
-                paper-tabs.bottom(selected='{{primaryViewIndex}}', hides, hidden$='[[!isZero(homeOrIframeViewIndex)]]')
-                    paper-tab Dashboard
-                    paper-tab(hidden$='[[!_devMode]]') Activity
-                aside#NamespaceSelector(hidden$='[[!_devMode]]')
-                    paper-dropdown-menu(label='Namespace')
-                paper-icon-button#Menu(icon='menu', on-click='toggleSidebar', hides, hidden$='[[!narrowMode]]')
-        neon-animated-pages.flex.layout.vertical(selected='[[homeOrIframeViewIndex]]', entry-animation='fade-in-animation', exit-animation='fade-out-animation')
-            neon-animated-pages#PrimaryView.flex.layout.vertical(selected='[[primaryViewIndex]]', entry-animation='fade-in-animation', exit-animation='fade-out-animation')
-                article#Dashboard
-                    paper-card.Getting-Started(heading='Getting Started', hidden$='[[!_devMode]]')
-                        template(is='dom-repeat', items='[[gettingStartedItems]]')
-                            paper-icon-item
-                                iron-icon(icon='[[item.icon]]', slot='item-icon')
-                                paper-item-body(two-line)
-                                    div [[item.text]]
-                                    aside(secondary) Lorem ipsum...
-                    paper-card.Quick-Links(heading='Quick Links')
-                        template(is='dom-repeat', items='[[quickLinks]]')
-                            article.link
-                                paper-item-body [[item.text]]
-                                paper-icon-button.button(icon='arrow-forward', alt='[[item.text]]', on-click='openQuickLink')
-                        article.link.more-coming
-                            paper-item-body More coming soon
-                            paper-icon-button.button(icon='arrow-forward', diabled)
-                article#Activity
-            iframe#PageFrame.flex(src='[[url]]')
+                header(hides, hidden$='[[hideToolbar]]')
+                    paper-tabs.bottom(selected='[[page]]', attr-for-selected='page')
+                        paper-tab(page='dashboard', link)
+                            a.link(tabindex='-1', href='/') DASHBOARD
+                        paper-tab(page='activity', link,hidden$='[[!_devMode]]')
+                            a.link(tabindex='-1', href='/activity') ACTIVITY
```

Comment:
The case doesn't match the mock:
![image](https://user-images.githubusercontent.com/5303018/53849689-88c35a80-3f6d-11e9-9563-16497a1dfbf6.png)

Also great job with simplifying the `neon-animatable-pages` with the `app-router`! 🎉

Reply:
Thanks. I think I got caught up viewing the GCP console Dashboard and Activity tabs which use all caps. 
![cczvgbouvu8](https://user-images.githubusercontent.com/6835846/53889185-f1edb100-3ff3-11e9-80a6-8a3f079bf449.png)

With respect to the namespace selector, ours has never included the rounded border shown in the mock, but I didn't make any changes to it in this PR.